Code/Engine/Core/World/Implementation/GameObject.cpp

```diff
@@ -895,51 +895,48 @@ void ezGameObject::PostMessageRecursive(const ezMessage& msg, ezTime delay, ezOb
   GetWorld()->PostMessageRecursive(GetHandle(), msg, delay, queueType);
 }
 
-void ezGameObject::SendEventMessage(ezEventMessage& msg, const ezComponent* pSenderComponent)
+void ezGameObject::SendEventMessage(ezMessage& msg, const ezComponent* pSenderComponent)
 {
-  ezHybridArray<ezComponent*, 4> eventMsgHandlers;
-  GetWorld()->FindEventMsgHandlers(msg, this, eventMsgHandlers);
-
-  if (eventMsgHandlers.IsEmpty() == false && pSenderComponent != nullptr)
+  if (auto pEventMsg = ezDynamicCast<ezEventMessage*>(&msg))
```

Comment:
I thought we'd just have overloads for ezMessage and ezEventMessage, then you wouldn't need to do dynamic casts.

Reply:
I didn't want to add another variant of this method because we already have so many. The dynamic cast is super cheap in comparison to the whole search for a message handler, so we should be fine in this case.